packages/cli/src/utils/ast.js

```diff
@@ -0,0 +1,139 @@
+// tools for modifiyng an AST
+
+const j = require('jscodeshift');
```

Comment:
this code is a little confusing, so let me know if the comments aren't clear enough. do you feel like you would be able to modify this later?

Reply:
Yeah, some serious stuff here! Playing around in AST Explorer helps me understand it a bit. The comments help too. I guess I've got the idea, so 👍